common/blockchain/src/manager.rs

```diff
@@ -282,6 +283,10 @@ impl<PK: crate::PubKey> ChainManager<PK> {
     ) -> Result<(), UpdateCandidateError> {
         self.candidates.update(&self.config, pubkey, stake)
     }
+
+    pub fn get_validators(&self) -> &[Validator<PK>] {
```

Comment:
Typically it’s more Rustic to not use ‘get_’ prefix for getters.
```suggestion
    pub fn validators(&self) -> &[Validator<PK>] {
```

Reply:
oh i see.